packages/paste-website/src/pages/components/toast/index.mdx

```diff
@@ -0,0 +1,362 @@
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/toast/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/toast"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-toast--neutral"
+  abstractUrl="https://share.goabstract.com/00bb0645-23b2-47aa-b57e-b0c6b1da9e8f"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+## About Toast
+
+Use a toast to communicate a direct response to a user based on their action. Using a toast lets users know that their action was acknowledged.
+
+Toasts appear over content on a page and stack vertically. All toasts have a close button so that users can dismiss any toasts that may be blocking underlying page content.
+
+Use a toast for one of these types of responses:
+
+- **Neutral or general response.** Let a user know that an asynchronous process has been triggered based on their action, especially if they have to wait for the process to finish before continuing a workflow. This could include a large download or a request that's been sent to a third party.
+- **Success.** Confirm a successful or completed action.
+- **Warning.** Help a user avoid a potential future issue that might happen based on their action. If the user can take an immediate action to avoid the issue, provide a way to do that. In most cases, a warning toast lets users know that their current action completed but may have triggered a related issue, or that their action can't be completed because of an issue that they need to fix outside their current flow.
+- **Error.** Communicate an issue that's preventing a user from completing their current action and can be fixed within their current flow.
+
+In general, use Toasts for response to user action. To decide whether you should use Toasts or another type of messaging, jump to [Toasts vs. Alerts vs. Inline messaging](#toast-vs-alerts-vs-inline-messaging).
+
+### Accessibility
+
+For [toasts that automatically time out](#toast-with-automatic-time-out), consider users with different reading speeds, vision levels, and literacy levels. We recommend giving users about 0.5 seconds per word. For short toasts at 5–10 words, that's about 2–4 seconds until you time-out a toast. For toasts at our [max suggested length of 140 characters](#composition-notes), that's about 30 seconds for about 55 English words. In this case, consider not making the toast automatically time out since the longer content length could also suggest that there's information worth persisting for the customer.
```

Comment:
```suggestion
For [toasts that automatically time out](#toast-with-automatic-time-out), consider users with different reading speeds, vision levels, and literacy levels (more details in the research insight below). We recommend giving users about 0.5 seconds per word. For short toasts at 5–10 words, that's about 2–4 seconds until you time-out a toast. For toasts at our [max suggested length of 140 characters](#composition-notes), that's about 30 seconds for about 55 English words. In this case, consider not making the toast automatically time out since the longer content length could also suggest that there's information worth persisting for the customer.
```